db/schema.rb

```diff
@@ -35,11 +56,13 @@
   end
```

Comment:
Why is this file updated in this PR?

Reply:
It was a residual error from earlier PRs (adding images), but now that PR is merged and we need that table in our schema.